app/services/pledge_form_generator.rb

```diff
@@ -0,0 +1,144 @@
+class PledgeFormGenerator
+  attr_reader :patient, :user, :case_manager_name
+
+  def initialize(user, patient, case_manager_name)
+    @patient = patient
+    @user = user
+    @case_manager_name = case_manager_name.to_s
```

Comment:
@feminismisawesome astutely noted that this should be `to_s`'d in the controller in https://github.com/colinxfleming/dcaf_case_management/pull/956/files#r108524143 .